src/popup/components/TransactionOverview.vue

```diff
@@ -32,11 +41,26 @@ export default {
           url: this.getExplorerPath(account.address),
         };
       },
-      isDexRecipient() {
-        return [...this.getDexContracts.router, ...this.getDexContracts.wae]
-          .includes(this.tx?.contractId);
-      },
     }),
+    isDexRecipient() {
+      return [...this.getDexContracts.router, ...this.getDexContracts.wae]
+        .includes(this.tx?.contractId);
+    },
+    getTxParty() {
```

Comment:
Maybe this could also be returned from the `transaction` composable as `transactionOwnershipAccount` or something similar?